intg-appletv/tv.py

```diff
@@ -194,6 +199,19 @@ def state(self) -> DeviceState | None:
         """Return the device state."""
         return self._state
 
+    @property
+    def output_devices(self) -> [str]:
+        """Return the list of possible selection (combinations) of output devices."""
+        return list(self._output_devices.keys())
+
+    @property
+    def output_device(self) -> str:
+        """Return the current selection of output devices."""
+        device_names = []
+        for device in self._output_device:
+            device_names.append(device.name)
+        return ",".join(sorted(device_names, key=str.casefold))
```

Comment:
Ok done, FYI I removed space initially because the webconfigurator dropdown list is not large enough to display the combination name. Maybe you could improve this ?